sdp/challenger_sdp.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 
 
-def setup(Q, T):
+def setup(Q, T):  # pragma: no cover
```

Comment:
I think we should test that each module has this function and that it returns `None`. Doing so would ensure that this function always exists.

Reply:
Good point! Will do!